### ISSUE TITLE
figure element source files / sept 4

### ISSUE THREAD

**Issue by wehooper** [IUBLibTech/newton_chymistry#34]
The figure element has an \@id attribute that supplies the name of the illustration file that won't work until those files are organized into the new framework.

Currently, the figure element is inserting some kind of line break/paragraph/div where there shouldn't be one. Do we want to do anything about this before the illustration files are available?

For examples, search for "stalk bearing" in Keynes 43 (ALCH00032). That's unique, and that paragraph contains two figures.

**Comment by Conal-Tuohy** [comment]
Where can I find the figure image files?

**Comment by wehooper** [comment]
We have two folders of images in the current (P4) html folder, images and img. 

The images folder contained icons and things for the website.

The img folder contains the illustrations organized by the document/catalog names, and a folder named unicode containing individual images of all the special characters that was useful when web fonts were just coming in, but may be less useful now. HTH

**Comment by Conal-Tuohy** [comment]
I have edited the P4-to-P5 convertor to interpret the `@id` convention described above and generate a P5 `graphic` element with a `url` attribute containing the URL of the gif image. I've added the directory tree of figure images for each MS, under a top-level directory called `figure`. The images therein are served up in response to request URIs with the pattern `/text/XXXX/figure/YYYYYYY.gif` 

The `@rend` attribute is propagated into the HTML as one of the values of the HTML `figure` element's `@class` attribute. e.g. a TEI `<figure rend="right">` will get you an HTML `<figure class="tei-figure rend-right">`, but I haven't yet added any CSS to make the `rend-right` do anything. I am going to make a list of all the `@rend` values and go through the list, when I'm otherwise done with the TEI-to-HTML conversion, so I'll deal with that then.
